R/tidy.R

```diff
@@ -497,9 +497,8 @@ competitiveness = function(map, rvote, dvote, .data=cur_plans()) {
     idxs = unique(as.integer(.data$draw))
     rvote = rlang::eval_tidy(rlang::enquo(rvote), map)
     dvote = rlang::eval_tidy(rlang::enquo(dvote), map)
-    rep(redist.competitiveness(plans=get_plans_matrix(.data)[, idxs, drop=FALSE],
-                               rvote=rvote, dvote=dvote),
-        each = attr(map, "ndists"))
+    redist.competitiveness(plans=get_plans_matrix(.data)[, idxs, drop=FALSE],
```

Comment:
see above. I feel like neither of these lines should have to change if we have 100% backwards compatibility?

If we have a good reason not to it's OK (major version increment) but we should add a message & add to NEWS